bitnami/grafana/templates/ingress.yaml

```diff
@@ -29,7 +29,7 @@ spec:
             {{- if eq "true" (include "common.ingress.supportsPathType" .) }}
             pathType: {{ .Values.ingress.pathType }}
             {{- end }}
-            backend: {{- include "common.ingress.backend" (dict "serviceName" (include "common.names.fullname" .) "servicePort" "http" "context" $)  | nindent 14 }}
+            backend: {{- include "common.ingress.backend" (dict "serviceName" (include "common.names.fullname" .) "servicePort" "{{ .Values.service.port }}" "context" $)  | nindent 14 }}
```

Comment:
This change is not necessary. Using the named port should be enough. Could you explain the issue you are having?

Reply:
Hi @javsalgar ,

Could you please explain why using the port named is enough in the servicePort. Due to this issue the port number in the ingress file is coming like this and sadly I am unable to open the rabbitmq management console:

![image](https://user-images.githubusercontent.com/14273153/107331051-5d8e0200-6ad8-11eb-997b-70abfb7f1d43.png)

But once I manually fixed this issue in my local machine and redeployed the ingress, it got changed and was able to open the management console after that:

![image](https://user-images.githubusercontent.com/14273153/107331196-8b734680-6ad8-11eb-9828-c252bad76eaa.png)


So could you please explain why this is not an issue in the Chart itself and there is no way to override this value.

Reply:
When setting an ingress rule, the named port is enough for Ingress for recognizing the service port. I just deployed it with ingress.enabled=true and could access without problems.

![image](https://user-images.githubusercontent.com/1133777/107343655-d5046700-6ac1-11eb-917d-6ddba0a53342.png)

Reply:
Hi @javsalgar ,

Sadly this setting is not working with the Azure Application Gateway, you probably tested it with the nginx-controller. If it is then can you please help how can I do the same with Azure App Gateway? 

P.S: Azure App Gateway does not allow text in the Port.

Reply:
Hi,

If that's the case, we should update all of our Ingress rules. Do you have a link where they confirm that incompatibility with named ports? I'll need it to forward it to the rest of the engineering team for evaluation.

Reply:
See if this link can help : [Link](https://github.com/Azure/application-gateway-kubernetes-ingress/blob/master/docs/annotations.md)

But we definitely need the updation as Azure Application Gateway is right away discarding the named port at the moment.

Reply:
I opened a ticket so they can confirm https://github.com/Azure/application-gateway-kubernetes-ingress/issues/1139

Reply:
okay thanks

Reply:
Still no answer. Let's give it some extra days before continuing with this.

Reply:
I pinged again

Reply:
Yeah, and I opened the same question [here](https://stackoverflow.com/questions/66204759/does-azure-application-gateway-ingress-controller-supports-named-ports) as well

Reply:
Let's do something to not block you. For the time being, I will accept this change in the chart, but I won't change it in the rest until we have a confirmation from Azure about this. Could you bump the minor version of the chart?

Reply:
@javsalgar could you please tell me the version of the chart which you bumped? We need to freeze the version in our project.

Reply:
Hi! We didn't bump the version, could you do it in this PR? The minor version would be required.